src/Mod/PartDesign/App/FeaturePipe.cpp

```diff
@@ -314,7 +374,7 @@ App::DocumentObjectExecReturn *Pipe::execute(void)
         //result.Move(invObjLoc);
         AddSubShape.setValue(result);
 
-        if (base.IsNull()) {
+        if (base.IsNull()) {


```

Comment:
You added a strange character

Reply:
That would be my emacs again. This isn't the first time it happened and unfortunately won't be the last.